torch/nn/modules/module.py

```diff
@@ -66,6 +66,8 @@ def __init__(self):
         self._backward_hooks = OrderedDict()
         self._forward_hooks = OrderedDict()
         self._forward_pre_hooks = OrderedDict()
+        self._state_dict_hooks = OrderedDict()
```

Comment:
I think you may need to conditionally create these in `__setstate__`

Reply:
Thanks! Fixed.